src/RoleObjectFactory.ts

```diff
@@ -147,6 +147,8 @@ function createAccessibilityObjectForRole(config: RoleObjectConfig) {
       );
       prevParent.removeChildAt(prevContainerIndex);
 
+      displayObject.accessible.removeAllChildren();
```

Comment:
Is this the right spot for this line?  It's inside the prevParent if branch, but I think this is independent of that check.  If container A is registered and has a child of DisplayObject B, then container A gets re-registered B should not refer to A in its parent field since A doesn't have B in its children array anymore.  But in that example it doesn't matter if A had a parent before being re-registered.

Reply:
good point - it would ostensibly "miss" the scenario of re-registering an unparented `displayObject` (as unlikely as that would seem)

Reply:
@mnewcomb314 re-tested with the change and I'm satisfied if you are =]

Reply:
Thanks! lgtm